### PR TITLE
WorkSpan connector new actions to support report API

### PR DESCRIPTION
**Pull request by WorkSpan-Engineering**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [Yes] I attest that the connector works and I verified by deploying and testing all the operations. 
- [Yes] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [Yes] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [Yes] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [Yes] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/WorkSpan/apiProperties.json [hunk 1]
```diff
@@ -124,58 +124,62 @@
       }
     },
     "iconBrandColor": "#c5dcff",
-    "publisher": "WorkSpan",
+	  "publisher": "WorkSpan",
```
Comment (sriyen-msft) [inline certified-connectors/WorkSpan/apiProperties.json:127]: @WorkSpan-Engineering Please adjust the spacing for the `"publisher"` property.

File: certified-connectors/WorkSpan/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1521,6 +1635,236 @@
...
+      },
+      "type": "object"
+    },
+    "ReportFilter": {
+      "properties": {
+        "negate": {
+          "type": "boolean",
+          "title": "",
+          "enum": [
+            "",
```
Comment (sriyen-msft) [inline certified-connectors/WorkSpan/apiDefinition.swagger.json:1821]: @WorkSpan-Engineering You have declared the property as boolean and defined the enum with a string value.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/WorkSpan/apiProperties.json (hunk 1 above)
Comment (WorkSpan-Engineering) [reply certified-connectors/WorkSpan/apiProperties.json:127]: ok

File: certified-connectors/WorkSpan/apiDefinition.swagger.json (hunk 2 above)
Comment (WorkSpan-Engineering) [reply certified-connectors/WorkSpan/apiDefinition.swagger.json:1821]: updating same

File: certified-connectors/WorkSpan/apiDefinition.swagger.json [hunk 3]
```diff
@@ -1521,6 +1635,235 @@
...
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "ReportFilter": {
+      "properties": {
+        "negate": {
+          "type": "boolean",
+          "title": "",
```
Comment (sriyen-msft) [inline certified-connectors/WorkSpan/apiDefinition.swagger.json:1819]: @WorkSpan-Engineering Empty property values can be removed until unless you have a specific reason to keep them.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**